bookkeeper-common/src/main/java/org/apache/bookkeeper/common/util/OrderedExecutor.java

```diff
@@ -187,10 +187,14 @@ public T build() {
     protected class TimedRunnable implements Runnable {
         final Runnable runnable;
         final long initNanos;
+        final String runnableString;
+        final Class<?> runnableClass;
 
         TimedRunnable(Runnable runnable) {
             this.runnable = runnable;
             this.initNanos = MathUtils.nowInNano();
+            this.runnableString = runnable.toString();
```

Comment:
toString() can be expensive; maybe skip this and only log the class?

Reply:
Nice suggestion. Updated @dlg99